ports/servoshell/desktop/minibrowser.rs

```diff
@@ -163,6 +166,84 @@ impl Minibrowser {
             .min_size(Vec2 { x: 20.0, y: 20.0 })
     }
 
+    /// Draws a browser tab, checking for clicks and returns an appropriate [EmbedderEvent]
+    /// Using a custom widget here would've been nice, but it doesn't seem as though egui
+    /// supports that, so we arrange multiple Widgets in a way that they look connected.
+    fn browser_tab(
+        ui: &mut egui::Ui,
+        label: &str,
+        selected: bool,
+        webview_id: TopLevelBrowsingContextId,
+    ) -> Option<EmbedderEvent> {
+        let old_item_spacing = ui.spacing().item_spacing;
+        let old_visuals = ui.visuals().clone();
+        ui.spacing_mut().item_spacing = egui::vec2(0.0, 0.0);
+
+        let visuals = ui.visuals_mut();
+        // Remove the stroke so we don't see the border between the close button and the label
+        visuals.widgets.active.bg_stroke.width = 0.0;
+        visuals.widgets.hovered.bg_stroke.width = 0.0;
+        // Now we make sure the fill color is always the same, irrespective of state, that way
+        // we can make sure that both the label and close button have the same background color
+        visuals.widgets.noninteractive.weak_bg_fill = egui::Color32::from_gray(40);
```

Comment:
Will this work for both light and dark themes? Is there some kind of color that works for both?

Reply:
Yeah, probably wouldn't. I've changed the code so that the colors are taken from the theme, that way it should work with most themes.